### PR TITLE
v2 tt5 demo

### PR DESCRIPTION
**Pull request by bph**
blueprint to 
- load wp beta
- login and use modern admin color
- empty site completely
- set pretty links
- Site title and description
- import demo content. 

This PR also updated the Blueprint Gallery.md to make it available from the Playground site, too

### CODE REVIEW

File: GALLERY.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Here's the list of all the community Blueprints submitted to this repository. Se
...
+| Demo of Twenty-Twenty-Five                                                    | Blueprint with demo content for the Twenty-Twenty-Five                                                                                                                                              | [@bph](https://github.com/bph) | • [Open in Playground](https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/tt5-demo/blueprint.json)<br>• [View source](https://github.com/wordpress/blueprints/blob/trunk/blueprints/tt5-demo/blueprint.json)<br>• [Edit](https://playground.wordpress.net/builder/builder.html?blueprint-url=https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/tt5-demo/blueprint.json)
```
Comment (brandonpayton) [inline GALLERY.md]: Should the word "Theme" follow "Blueprint with demo content for the Twenty-Twenty-Five"?

**Review by brandonpayton — APPROVED** [review]
This looks good. I left a comment asking whether the description should say "theme" somewhere, but that's it. Thank you!

File: GALLERY.md (hunk 1 above)
Comment (bph) [reply GALLERY.md]: Yes, somehow my copy/pasta got truncated. Thanks for spotting this @brandonpayton. Fixed now.

Comment (brandonpayton) [reply GALLERY.md]: It happens :)  Thanks!

**Pull request merged**